nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/test/java/org/apache/nifi/processors/kudu/TestPutKudu.java

```diff
@@ -412,6 +413,7 @@ private void buildPartialRow(Long id, String name, Short age) {
             kuduSchema.newPartialRow(),
             new MapRecord(schema, values),
             schema.getFieldNames(),
+                true,
```

Comment:
I think setting `lowercaseFields` to `true` is a bit misleading. At the moment this only works because
1. All the `fieldNames` in the test are already in lower case.
2. There are no assertions that the `buildPartialRow` actually did set the values correctly.

Reply:
I'm not actually modifying the schema of the row, I'm only converting the column name to lowercase before it looks up the index in the table schema, from there, the column is referenced by it's index rather than the column name so I don't think adding an assertion that row's column names are lowercase makes sense for testing this. 
Since Kudu is case sensitive, testing that an uppercase column name returns a value != -1 when retrieving the index and lowercaseFields == true tests the desired functionality which is partially why I added an exception if column index = -1

Reply:
Added a unit test with uppercase schema.